### ISSUE TITLE
Cohort labels are long for some countries

### ISSUE THREAD

**Issue by alimanfoo** [anopheles-genomic-surveillance/selection-atlas#60]
E.g.:

![image](https://github.com/anopheles-genomic-surveillance/selection-atlas/assets/703554/a85058ff-e829-4c84-97df-cf7cefc93c02)

Maybe we should use the ISO alpha-2 country codes in the cohort labels instead of the full country names.

**Comment by sanjaynagi** [comment]
Happy to change but I personally think this is OK as is - mainly because some of the ISO codes are not super obvious which country they refer to if you arent familiar with them...

**Comment by alimanfoo** [comment]
Yeah on second thought I think this probably fine too, let's leave as is.
